### DISCUSSION TITLE
How to select which 3D volume of a 4D volume is displayed via pyAnatomist command?

### DISCUSSION THREAD

**Discussion by augustin-ogier** [brainvisa/brainvisa.github.io#153] [Q&A]
Hello,
I'm using pyAnatomist to script the display of a 4D cardiac volume.
I couldn't find in the documentation (sorry if it was easy to find) how to change the 3D volume displayed?
Can you please tell me which command I should send to Anatomist to choose the volume **t** to display?

I thought it would be something like `a.execute('Camera', windows=[w], position=[x, y, z, `**t**`])` but that seems to only affect the cursor position and not the display.

Thanks a lot!

**Comment by denisri** [comment]
Hi,
The correct parameter is `cursor_position`, not `position`, that's why it's not working.
Otherwise you can use the window method `w.camera(cursor_position=[x, y, z, t])`, which is exactly the same but with python parameters checking (it would have thrown an error, the old command system `a.execute()` does not raise errors).
Denis

**Reply by augustin-ogier** [reply]
Hi,
Exactly what I wanted, it works perfectly!
Thank so much for your quick reply!

Augustin
